import/Ubuntu/Content/ResponsiveGridView.qml

```diff
@@ -56,7 +56,7 @@ Item {
         }
 
         onModelChanged: {
-            clip = parent.height != contentHeightForRows(Math.ceil(model.count / columns))
+            clip = model ? parent.height != contentHeightForRows(Math.ceil(model.count / columns)) : false
```

Comment:
This one can be simplified as `model && parent.height != contentHeightForRows(Math.ceil(model.count / columns))` instead.

Reply:
Oh you are right, thanks for pointing out.

Reply:
Great. Can you squash the second commit into the first using `git rebase -i HEAD~2` and presesrve just the first commit message? Thanks!

Reply:
Well that was a mess LOL
After many failed attempts, I think I basically redone the changes :sweat_smile: 
Let me know if it's messed up and we need to create a new PR......